README.md

```diff
@@ -1,369 +1,168 @@
-
-#### From Foreign to Home Network transaction:
-To burn and send back to Home Network (ERC20 to POA20):
-- Specify an Amount
-- Click the switch button and click the <- arrow button to send
-- Confirm transaction on MetaMask
-- The address that you use to send from, the same address will be used to receive the coin on a Home chain
-
-The amount sent must be within the daily limits provided by the contracts. When the transaction is validated, the user should expect to see a Withdrawal Event on the Foreign Network (right-side - Ethereum Foundation). After some time, Validators submit signatures emitting SignedForWithdrawal event to the Foreign Network. Once the required number of signatures is reached, CollectedSignatures event is emitted on the Foreign Network. The ERC20 tokens are burned on the Foreign Network. This process generates a signed message which Validators submit on the Home Network. The user automatically receives their Native POA Coin.
-
-# Storage of funds
-When you send POA on Home Network(POA.network), it gets stored and locked into a smart contract on POA.network.
-When you send POA20 on Foreign Network (Ethereum Mainnet), it is automatically burnt and reduced from totalSupply, therefore it's not stored anywhere right after a sender initiated a transfer to Home network. Once the transfer is relayed, locked funds on Home network are withdrawn to the sender automatically.
-
-# Responsibilities and roles of the bridge:
-- Administrator Role(representation of a multisig contract):
-  - add/remove validators
-  - set daily limits on both bridges
-  - set maximum per transaction limit on both bridges
-  - set minimum required signatures from validators in order to relay a user's transaction
-- Super Administrator for Validator Contract (representation of a multisig contract):
-  - upgrade contracts in case of vulnerability
-- Super Administrator for Bridge Contract (representation of a multisig contract):
-  - upgrade contracts in case of vulnerability
-- Validator Role :
-  - provide 100% uptime to relay transactions
-  - listen for Deposit events on Home bridge to mint erc20 token on Foreign bridge
-  - listen for Withdraw events on Foreign bridge to unlock funds on Home Bridge
-- User role:
-  - sends POA coins to Home bridge in order to receive ERC20 token on Foreign Bridge using the same address to send/receive
-  - sends ERC20 POA20 token on Foreign Bridge in order to receive POA coins on Home Bridge using the same address to send/receive
-
-## Dependencies
-
-- [poa-bridge-contracts](https://github.com/poanetwork/poa-bridge-contracts/tree/0.1)
+### User Transactions
+- Connect to the network you want to transfer coins from using a web3 wallet such as Nifty Wallet or MetaMask. This can be the Home or Foreign network. The wallet must be funded to include amount to transfer and any gas costs.
```

Comment:
1. The suggestion is to re-phrase: `The wallet must be funded to cover any gas costs and, in case of `native-to-erc20` bridge mode, to include amount to transfer.`
2. Also, for `erc20-to-erc20` mode it makes sense to mention that account must own tokens linked with the particular bridge.